src/utils/logger.ts

```diff
@@ -0,0 +1,31 @@
+import winston from 'winston'
```

Comment:
Not too familiar with Winston but I'll look into it!

Reply:
I just used it because it was mentioned in the discordjs.guide website. We could look into other loggers, which is why i implemented a ILogger interface.